js/samples/quickstart-java/src/main/webapp/resources/helpers.js

```diff
@@ -0,0 +1,21 @@
+function getTokenAndSubdomainAsync() {
```

Comment:
I don't see that this fcn sets the subdomain.

Reply:
done, ranamed to getTokenAsync